soh/soh/Enhancements/bootcommands.c

```diff
@@ -27,6 +27,11 @@ void BootCommands_Init()
     CVar_RegisterS32("gHudColors", 1); //0 = N64 / 1 = NGC / 2 = Custom
 	CVar_RegisterS32("gInvertYAxis", 1);
     CVar_RegisterS32("gTrailDuration", 4); // 4 = Default trail duration
+    if (ResourceMgr_IsGameMasterQuest()) {
+        CVar_SetS32("gRandomizer", 0);
+    } else {
+        CVar_RegisterS32("gRandomizer", 0);
```

Comment:
we probably don't actually need this register call but it doesn't hurt to have it